### ISSUE TITLE
How to avoid this kind of thing?

### ISSUE THREAD

**Issue by filipecosta01** [recruit-lifestyle/FloatingView#14]
Seems like including a default layout to be inflated causes this weird behavior, the trash shaded area is  jumping... Idk how to solve it, as I really need the message prompted as the image attached.

I'm using the configurations like the sample app (chat heads one), but inflating this layout instead of only the imageview:

```
<?xml version="1.0" encoding="utf-8"?>
   <LinearLayout
    xmlns:android="http://schemas.android.com/apk/res/android"
    xmlns:tools="http://schemas.android.com/tools"
    android:orientation="horizontal"
    android:layout_width="wrap_content"
    android:layout_height="wrap_content"
    android:background="@drawable/shape_tin_message"
    android:padding="16dp">

    <RoundedImageView
        xmlns:app="http://schemas.android.com/apk/res-auto"
        android:layout_gravity="center"
        android:scaleType="centerCrop"
        android:layout_width="20dp"
        android:layout_height="20dp"
        app:oval="true"
        android:src="@drawable/ic_launcher"/>

    <TextView
        android:id="@+id/tin_message"
        android:layout_width="match_parent"
        android:layout_height="wrap_content"
        android:textColor="@color/white"
        android:textStyle="bold"
        android:textSize="16sp"
        android:layout_gravity="center_vertical"
        android:layout_marginStart="5dp"
        android:textAppearance="?android:textAppearanceMedium"
        tools:text="My TIN # is: 1125198411"/>

</LinearLayout>
```

![image](https://cloud.githubusercontent.com/assets/2905403/15516309/932bcbda-21c8-11e6-871b-ef0b06588075.png)

**Comment by YoshihideSogawa** [comment]
The bug did not happen... (Nexus5)
If there is a problem, please look [here](https://github.com/recruit-lifestyle/FloatingView/blob/master/library/src/main/java/jp/co/recruit_lifestyle/android/floatingview/TrashView.java#L857).

**Comment by YoshihideSogawa** [comment]
Fix [Ver 2.2](https://github.com/recruit-lifestyle/FloatingView/releases/tag/2.2)
